include/crocoddyl/multibody/residuals/contact-friction-cone.hxx

```diff
@@ -13,23 +13,17 @@ namespace crocoddyl {
 template <typename Scalar>
 ResidualModelContactFrictionConeTpl<Scalar>::ResidualModelContactFrictionConeTpl(
     boost::shared_ptr<StateMultibody> state, const pinocchio::FrameIndex id, const FrictionCone& fref,
-    const std::size_t nu)
-    : Base(state, fref.get_nf() + 1, nu, true, true, true), id_(id), fref_(fref) {
-  if (static_cast<pinocchio::FrameIndex>(state->get_pinocchio()->nframes) <= id) {
```

Comment:
We should add this safety check back

Reply:
Thank you for catching it. I guess this happened during rebasing this code.